[2022-11-29T06:20:02.031+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.task_6 manual__2022-11-29T06:19:39.257639+00:00 [queued]>
[2022-11-29T06:20:02.042+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.task_6 manual__2022-11-29T06:19:39.257639+00:00 [queued]>
[2022-11-29T06:20:02.043+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T06:20:02.044+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-29T06:20:02.046+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T06:20:02.063+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): task_6> on 2022-11-29 06:19:39.257639+00:00
[2022-11-29T06:20:02.072+0000] {standard_task_runner.py:55} INFO - Started process 1446 to run task
[2022-11-29T06:20:02.076+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Can_I_change_name_to_this', 'task_6', 'manual__2022-11-29T06:19:39.257639+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmptwm0r3q3']
[2022-11-29T06:20:02.078+0000] {standard_task_runner.py:83} INFO - Job 36: Subtask task_6
[2022-11-29T06:20:02.129+0000] {task_command.py:376} INFO - Running <TaskInstance: Can_I_change_name_to_this.task_6 manual__2022-11-29T06:19:39.257639+00:00 [running]> on host 68e922a5ec23
[2022-11-29T06:20:02.199+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Can_I_change_name_to_this
AIRFLOW_CTX_TASK_ID=task_6
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T06:19:39.257639+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T06:19:39.257639+00:00
[2022-11-29T06:20:02.202+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-29T06:20:02.205+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/tasks/data_cleansing_basic_stat.py']
[2022-11-29T06:20:02.229+0000] {subprocess.py:86} INFO - Output:
[2022-11-29T06:20:04.113+0000] {subprocess.py:93} INFO - {"message": "now perform data cleansing on lol_basic_stat", "time": "2022-11-29T06:20:04.109530"}
[2022-11-29T06:20:04.130+0000] {subprocess.py:93} INFO - {"message": "successfully perform data cleansing on lol_basic_stat", "time": "2022-11-29T06:20:04.130253"}
[2022-11-29T06:20:04.142+0000] {subprocess.py:93} INFO - {"message": "Now putting dataframe into pg and redshift", "time": "2022-11-29T06:20:04.133854"}
[2022-11-29T06:20:04.838+0000] {subprocess.py:93} INFO - {"message": "successfully put lol_basic_stat to pg and redshift", "time": "2022-11-29T06:20:04.833680"}
[2022-11-29T06:20:04.851+0000] {subprocess.py:93} INFO - {"message": "put lol_basic_stat to pg and redshift have used 0.8665878339998017s", "time": "2022-11-29T06:20:04.836109"}
[2022-11-29T06:20:05.231+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-29T06:20:05.371+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Can_I_change_name_to_this, task_id=task_6, execution_date=20221129T061939, start_date=20221129T062002, end_date=20221129T062005
[2022-11-29T06:20:05.600+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-29T06:20:05.796+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
